### PR TITLE
Add recipe for evil-tutor-sc

### PR DESCRIPTION
**Pull request by clsty**
### Brief summary of what the package does

[evil-tutor](https://github.com/syl20bnr/evil-tutor) in Simplified Chinese.

### Direct link to the package repository

https://github.com/clsty/evil-tutor-sc

### Your association with the package

Author.
- `tutor-sc.txt` is translated from `tutor.txt` of [evil-tutor](https://github.com/syl20bnr/evil-tutor).
This part is also the main effort worked on this package by myself.
- `evil-tutor-sc.el` is directly from `evil-tutor-ja.el` of [evil-tutor-ja](https://github.com/kenjimyzk/evil-tutor-ja).

### Relevant communications with the upstream package maintainer

**None needed**

### Checklist

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] (o) I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] (o) I've used `M-x checkdoc` to check the package's documentation strings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

(o): Not needed because the only elisp file `evil-tutor-sc.el` is directly modified from `evil-tutor-ja.el` of [evil-tutor-ja](https://github.com/kenjimyzk/evil-tutor-ja) , which is already in MELPA .

In other words, `evil-tutor-ja` already did it "for" this project.

P.S. modification on `evil-tutor-ja.el` : ONLY 2 things below:
- Metadata (Author, Keywords, Created date, URL) modified accordingly (which surely affects nothing in functionality).
- "ja" replaced to "sc"(naturally because this is Simplified Chinese version, not Japanese version; won't affect any functionality either).

### CODE REVIEW

**Comment by clsty** [comment]
@riscy Hi, may you have a look at this when you have time?

I guess it won't cost you much time,
because `evil-tutor-sc.el` of this project was directly copied from a MELPA package with only minor modifications (main effort is on `tutor-sc.txt` ).

**Comment by riscy** [comment]
Sure - I'm mostly here on weekends (primarily Sundays these days); apologies for the delay.

Thanks for this submission.  One quick note I'll make is that even though other packages on MELPA may not meet the checklist requirements, that doesn't mean I'll automatically waive those requirements in my reviews.   (Otherwise the bar would really be quite low, to be honest.)  Here's a quick first pass --

`evil-tutor-sc.el` with package-lint 20230605.1550:
```
2 issues found:
1:0: warning: The package summary is too long. It should be at most 60 characters.
51:0: error: "evil-tutor-mode" doesn't start with package's prefix "evil-tutor-sc".
```

`evil-tutor-sc.el` with [melpazoid](https://github.com/riscy/melpazoid):
```
- evil-tutor-sc.el#L63: It's safer to sharp-quote function names; use `#'`
```

**Comment by clsty** [comment]
> 

@riscy All fixed accordingly. Sorry for my assumption that all MELPA packages should meet the checklist requirements. Thanks for providing first pass information.

**Comment by riscy** [comment]
Looks good - I notice you've got commented-out copies of `evil-tutor-goto-previous-lesson`, `evil-tutor-goto-next-lesson`, and `evil-tutor--find-first-working-file` at the bottom of your source file (and also some commented-out code around `evil-tutor-mode-map` further up).  You should definitely clean all of that up.  Also "restore last working fil" in your `;;; Commentary` should be "working file".

But these fixes aren't blockers.  Merging now -- and welcome to MELPA. :)
